src/makeRobot.js

```diff
@@ -37,8 +37,64 @@
  *
  * @return {object}
  */
+
 function makeRobot(name, wheels, version) {
-  // write code here
+  return {
+    robotName: name,
+    wheelsCount: wheels,
+    chipVersion: version,
```

Comment:
You can just leave the names of variables the way they are
```suggestion
  return {
    name,
    wheels,
    version,
```